definitions/variable/technology/technologies.yaml

```diff
@@ -55,10 +63,18 @@
     description: Capital cost for freight {Transport}
     unit: [EUR_2020/tkm/yr, USD_2010/tkm/yr]
     skip-region-aggregation: true
+- Fixed Cost|Transportation|Freight|Rail:
+    description: Capital cost for freight rail-based transportation technologies
+    unit: [EUR_2020/tkm/yr, USD_2010/tkm/yr]
+    skip-region-aggregation: true
 - Fixed Cost|Transportation|Passenger|{Transport}:
     description: Capital cost for passenger {Transport}
     unit: [EUR_2020/pkm/yr, USD_2010/pkm/yr]
     skip-region-aggregation: true
+- Fixed Cost|Transportation|Passenger|Rail:
```

Comment:
Why is this not part of the {Transport} tag-list?

Reply:
This is where the second option you described in #162 would come in handy.
For now it's good to keep as is here.
Maybe open an issue to revisit this if or when that feature gets implemented.